app/src/main/java/com/onboarding/parkingsystemjava/mvp/view/SetParkingView.java

```diff
@@ -0,0 +1,34 @@
+package com.onboarding.parkingsystemjava.mvp.view;
+
+import android.widget.TextView;
+import android.widget.Toast;
+
+import com.onboarding.parkingsystemjava.R;
+import com.onboarding.parkingsystemjava.fragment.SetParkingLotsDialog;
+import com.onboarding.parkingsystemjava.mvp.contract.SetParkingContract;
+
+import java.lang.ref.WeakReference;
+
+public class SetParkingView implements SetParkingContract.SetParkingView {
+    private WeakReference<SetParkingLotsDialog> fragmentRef;
+
+    public SetParkingView(SetParkingLotsDialog fragment) {
```

Comment:
pass here the binding as we do in the activity

Reply:
done